netbout-web/src/main/java/com/netbout/dynamo/DyMessages.java

```diff
@@ -202,7 +204,14 @@ public long unread() throws IOException {
             )
             .where(DyFriends.HASH, Conditions.equalTo(this.bout))
             .where(DyFriends.RANGE, Conditions.equalTo(this.self))
-            .iterator().next();
+            .iterator();
+        if (!iterator.hasNext()) {
+            throw new RsFailure(
```

Comment:
@dmzaytsev if we will return 0 than kicked user will stay in the bout, post messages and etc. `RsFailure` will redirect him to home page where he wan't see bout.